joplin/api/schema.py

```diff
@@ -359,6 +359,49 @@ class Meta:
         interfaces = [graphene.Node]
 
 
+class DepartmentPageTopServiceNode(DjangoObjectType):
+    service_page = graphene.Field(ServicePageNode)
+    information_page = graphene.Field(InformationPageNode)
+    guide_page = graphene.Field(GuidePageNode)
+
+    def resolve_service_page(self, info):
+        service_page = None
+        # TODO: don't catch everything
+        try:
+            service_page = ServicePage.objects.get(id=self.page_id)
+        except:
```

Comment:
What sorts of exceptions or errors might we expect here?

Reply:
I'm not sure exactly what the errors are, but the issue comes into play when we make a query that tries to get all of the page types. We aren't sure which page type goes to which ID, so this allows us to have a GraphQL query where we get informationPage, servicePage, and guidePage and just have null for the types a page isn't without any errors